### PR TITLE
fixes - sccp-350

### PR DESCRIPTION
**Pull request by kaleb-keny**

### CODE REVIEW

File: omnibus-arbitrum-sepolia.toml [hunk 1]
```diff
@@ -238,6 +238,14 @@ description = "1 hour"
 defaultValue = "<%= String(1 * 3600) %>"
 description = "1 hour"
 
+[setting.USDe_staleness_tolerance]
+defaultValue = "<%= String(1 * 3600) %>"
+description = "1 hour"
+
+[setting.sol_staleness_tolerance]
```
Comment (noisekit) [inline omnibus-arbitrum-sepolia.toml:245]: can we keep casing consistent
```suggestion
[setting.SOL_staleness_tolerance]
```

**Review by noisekit — APPROVED** [review]

File: tomls/omnibus-arbitrum-sepolia/spot/dai.toml [hunk 2]
```diff
@@ -17,7 +17,7 @@ args = [
     "<%= extras.synth_dai_market_id %>",
     "<%= extras.dai_oracle_id %>",
     "<%= extras.dai_oracle_id %>",
-    "<%= settings.dai_staleness_tolerance %>",
+    "<%= settings.big_cap_strict_staleness_tolerance %>",
```
Comment (noisekit) [inline tomls/omnibus-arbitrum-sepolia/spot/dai.toml:20]: @kaleb-keny updated for all spot markets

Comment (kaleb-keny) [reply tomls/omnibus-arbitrum-sepolia/spot/dai.toml:20]: i'll setup stuff my end to keep on eye on these 😅

Comment (kaleb-keny) [reply tomls/omnibus-arbitrum-sepolia/spot/dai.toml:20]: thanks for pointing it out

**Pull request merged**